### ISSUE TITLE
not Syncing 

### ISSUE THREAD

**Issue by mtrsmtrs** [Chia-Network/chia-blockchain#14986]
### What happened?

There are two days left on Syncing 3,474,223/3,477,798 and it doesn't change. Please help.

### Version

1.7.1

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by MumfMeisterT** [comment]
Does restarting the client get it to start again? Could you provide us with debug logs?

**Comment by shaneo257** [comment]
also, have you got enough free space on the drive where the DB is stored

**Comment by mtrsmtrs** [comment]
> آیا راه اندازی مجدد باعث می شود که دوباره شروع شود؟ آیا می توانید گزارش های اشکال زدایی را در اختیار ما قرار دهید؟

Yes, everything works normally.
and restarts
[debug.log](https://github.com/Chia-Network/chia-blockchain/files/11175692/debug.log)

[debug.log](https://github.com/Chia-Network/chia-blockchain/files/11175701/debug.log)

**Comment by mtrsmtrs** [comment]
> همچنین، آیا فضای خالی کافی در درایوی دارید که DB در آن ذخیره شده است؟

Currently, the space occupied by the db folder is 106 GB and there is still 311 GB of free space left.

**Comment by mtrsmtrs** [comment]
Hello .
I tried all the ways that were simple, but everything was fine and there was no change. In the end, I only saved the contents of the db folder and then changed Windows and installed version 1.7.0 and let my system work for half an hour to make sure the problem was fixed, after that I completely closed the program with the task manager and I replaced the db folder with a new folder so that I don't have to wait a few days for it to become a full node and with DNS 185.51.200.2 178.22.122.100
I ran the program and it was successful. I will complete the steps here so that others will not be bothered for a long time like me. Good luck
